examples/log_to_file/src/main.rs

```diff
@@ -0,0 +1,52 @@
+// cargo install cargo-edit
+// cargo add log4rs log
+
+use log::LevelFilter;
+use log::{debug, error, info, trace, warn};
+use log4rs::append::console::{ConsoleAppender, Target};
+use log4rs::append::file::FileAppender;
+use log4rs::config::{Appender, Config, Root};
+use log4rs::encode::pattern::PatternEncoder;
+use log4rs::filter::threshold::ThresholdFilter;
+
+fn main() {
+    // You probably want to pass this as a parameter.
```

Comment:
This seems redundant, if you think a comment is important here, maybe it needs to be more explanatory?

Reply:
Yeah, I think we can remove this and https://github.com/sfackler/log4rs/pull/97#discussion_r335284080